src/debug.c

```diff
@@ -1823,10 +1828,10 @@ void logCurrentClient(void) {
     int j;
 
     serverLogRaw(LL_WARNING|LL_RAW, "\n------ CURRENT CLIENT INFO ------\n");
-    client = catClientInfoString(sdsempty(),cc);
+    client = catClientInfoStringGeneric(sdsempty(),cc,server.hide_client_info);
     serverLog(LL_WARNING|LL_RAW,"%s\n", client);
     sdsfree(client);
-    for (j = 0; j < cc->argc; j++) {
+    for (j = 0; j < clientArgsToShow(cc); j++) {
```

Comment:
if we hide the arguments, let's at least print the `argc` instead, and maybe a bold "redacted" message.